lib/ansible/modules/crypto/openssl_csr.py

```diff
@@ -200,6 +200,51 @@
         type: bool
         default: no
         version_added: "2.8"
+    create_subject_key_identifier:
+        description:
+            - Create the Subject Key Identifier from the public key.
+            - Note that this is only supported if the C(cryptography) backend is used!
+        type: bool
+        default: no
+        version_added: "2.9"
+    subject_key_identifier:
+        description:
+            - The subject key identifier as a hex string, where two bytes are separated by colons.
+            - "Example: C(00:11:22:33:44:55:66:77:88:99:aa:bb:cc:dd:ee:ff:00:11:22:33)"
+            - Note that this option can only be used if I(create_subject_key_identifier) is C(no).
+            - Note that this is only supported if the C(cryptography) backend is used!
```

Comment:
May be worth noting that all of the added values are applicable only to creating a self signed certificate, or using the openssl_certificate module with self_signed or ownca - they wouldn't ever be applicable to a CSR submitted elsewhere (at least to my knowledge).

Reply:
I added a disclaimer in 2c03b5d68ce59eb1ad321069a52dbba9a950bb2f